stable/nginx-ingress/templates/controller-metrics-service.yaml

```diff
@@ -19,7 +19,9 @@ metadata:
     release: {{ .Release.Name }}
   name: {{ template "nginx-ingress.controller.fullname" . }}-metrics
 spec:
+{{- if not .Values.controller.metrics.service.omitClusterIP }}
```

Comment:
Instead of a new value that can be set lets have these check if the current value of clusterIP contains anything.
```
{{- if .Values.controller.metrics.service.clusterIP }}
```

https://github.com/helm/helm/blob/master/docs/chart_template_guide/control_structures.md#ifelse

Reply:
Hey @ChiefAlexander 
Of course that's the obvious solution but it's not usable here. This is all covered in https://github.com/helm/charts/issues/9227

This was tried in https://github.com/helm/charts/pull/7933 and reverted in https://github.com/helm/charts/pull/10993

All users who installed the chart between version 0.31.0 and 1.2.2 cannot upgrade.

Reply:
This whole thing is a disaster. Add a section in the readme to document this and how users need to use it to upgrade from those versions.

Reply:
OK done